### ISSUE TITLE
Closed codelist errors appear in both 'validation errors' and 'codelist validation errors'

### ISSUE THREAD

**Issue by duncandewhurst** [open-contracting/cove-ocds#110]
For example, the error with the procurementMethod codelist appears twice:

![image](https://user-images.githubusercontent.com/19265814/43996700-353c7eb0-9d96-11e8-8e8b-9f5d8c3903ac.png)

Could we get add the additional information which appears in the 'validation errors' section (the error count and location of first 3 errors) to the information in the 'codelist validation errors' section and just show the error in the codelist section?

**Comment by jpmckinney** [comment]
Agreed that it would be better to have codelist errors in validation_errors. However, tools expect these errors in additional_closed_codelist_values, so they are left there.

**Comment by jpmckinney** [comment]
Actually, closed codelists errors DO appear in both - it just has to be a closed codelist that is also an array, like `tag` in OCDS 1.1.

Extensions with arrays using closed codelists are:

* selectionCriteria `appliesTo`
* participationFee `type`
* bid `countriesOfOrigin`

**Comment by jpmckinney** [comment]
Test data

String closed codelist:

`{"version":"1.1","releases":[{"tender":{"procurementMethod":"bad"}}]}`

`{"version":"1.1","records":[{"releases":[{"id":"1","tender":{"procurementMethod":"bad"}}]}]}`

Array closed codelist:

`{"version":"1.1","releases":[{"tag":["bad"]}]}`

`{"version":"1.1","records":[{"releases":[{"id":"1","tag":["bad"]}]}]}`
